feathr_project/feathr/client.py

```diff
@@ -459,15 +459,16 @@ def _construct_redis_key(self, feature_table, key):
             key = self._COMPOSITE_KEY_SEPARATOR.join(key)
         return feature_table + self._KEY_SEPARATOR + key
 
-    def _str_to_bool(self, s):
+    def _str_to_bool(self, s, variable_name = None):
         """Define a function to detect convert string to bool, since Redis client sometimes require a bool and sometimes require a str
         """
         if s == 'True' or s == True:
```

Comment:
Suggest support string true and false here. (lowercase). Right now if user defines with 
ssl_enabled: true or ssl_enabled: TRUE, the config might not be picked up correctly.

On a side note, should redis init be decoupled from feathr client init? I am having concern to see redis init happen in a critical path.

Reply:
good point